### ISSUE TITLE
Document bookkeeper shell command changes

### ISSUE THREAD

**Issue by sijie** [apache/bookkeeper#914]
There are bunch of command changes in bookie shell. New commands are added, some options are deprecated from existing commands. It would be good to document all the shell command changes and incorporate them in 4.7 release notes.

**Comment by jiazhai** [comment]
@sijie @reddycharan, how about also update http module according to these issues?

**Comment by sijie** [comment]
@jiazhai good point. can you file a separate issue for it? so we remember adding them to http module as well.
